solarforecastarbiter/io/reference_observations/srml.py

```diff
@@ -36,17 +46,38 @@
 logger = logging.getLogger('reference_data')
 
 
+def adjust_site_parameters(site):
```

Comment:
looks like nearly a copy of the `pvdaq` function. should we refactor into common?

Reply:
Yes.  Minimal changes would be to move the meat to common with a sitefile kwarg then just call that from pvdaq/srml with the right sitefile